tests/xtro-sharpie/common.pending

```diff
@@ -77,6 +77,15 @@
 !missing-selector! SKVideoNode::initWithFileNamed: not bound
 !missing-selector! SKVideoNode::initWithURL: not bound
 
+## Do we really need these static factory methods, instead of the nice construcotrs we have in C#?
+## Only reason Apple exposed these is because [[foo alloc] init] is annoying
```

Comment:
I agree the *positive* value of adding them seems low. However the *negative* value of ignoring them is higher, including:

* it's additional work to make sure there are identical `init*` methods (that could be automated but it's not);

* it's additional maintenance (on xtro) to track future changes, e.g. new ones could easily be added, leading to inconsistent API (that's a bit harder to automate as it mix intro and xtro data);

* it's reduce the helpfulness of monodoc where customers can search on selectors to find the corresponding managed API (again maybe this could be done in another way, but we don't have such tools).

The above goes away by adding the bindings and the linker will removed unused ones so their duplication is not even a big deal :)